### PR TITLE
spread: add external:ubuntu-core-24-64

### PR DESCRIPTION
**Pull request by valentindavid**
This is needed by core-base CI.

### CODE REVIEW

**Comment by valentindavid** [comment]
Needed by snapcore/core-base#89

File: tests/lib/external/snapd-testing-tools/tools/os.query [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```
Comment (mvo5) [inline tests/lib/external/snapd-testing-tools/tools/os.query:3]: This looks like a debug  leftover

**Review by mvo5 — COMMENTED** [review]
Thank you, this looks fine except the one debug leftover. AFAICT we could promote this out of draft and merge, or is there a downside of doing this?

File: tests/lib/external/snapd-testing-tools/tools/os.query [hunk 2]
```diff
@@ -60,6 +62,16 @@ is_core22() {
     fi
 }
 
+is_core24() {
```
Comment (sergiocazzolato) [inline tests/lib/external/snapd-testing-tools/tools/os.query:65]: Hi, this needs to be included in the snapd-testing-tools project

**Review by sergiocazzolato — CHANGES_REQUESTED** [review]
just a comment about moving part of the change to snapd-testing-tools project

File: tests/lib/external/snapd-testing-tools/tools/os.query (hunk 2 above)
Comment (valentindavid) [reply tests/lib/external/snapd-testing-tools/tools/os.query:63]: https://github.com/snapcore/snapd-testing-tools/pull/38

File: tests/lib/external/snapd-testing-tools/tools/os.query (hunk 1 above)
Comment (valentindavid) [reply tests/lib/external/snapd-testing-tools/tools/os.query]: I have removed it.

File: tests/lib/prepare.sh [hunk 3]
```diff
@@ -58,6 +58,9 @@ ensure_jq() {
     elif os.query is-core22; then
         snap install --devmode --edge jq-core22
         snap alias jq-core22.jq jq
+    elif os.query is-core24; then
+        snap install --devmode --edge jq-core22
```
[the remark this answers is not in the document]
Comment (valentindavid) [reply tests/lib/prepare.sh:63]: Done

**Review by mvo5 — APPROVED** [review]
Thank you

File: tests/lib/external/snapd-testing-tools/tools/os.query (hunk 2 above)
Comment (sergiocazzolato) [reply tests/lib/external/snapd-testing-tools/tools/os.query:63]: I think you just need to merge master because it is on master

**Review by sergiocazzolato — COMMENTED** [review]
Hi, I left a comment about os.query is-core24

File: tests/lib/external/snapd-testing-tools/tools/os.query (hunk 2 above)
Comment (valentindavid) [reply tests/lib/external/snapd-testing-tools/tools/os.query]: Done.

**Review by sergiocazzolato — APPROVED** [review]
Nice, thanks for this

**Pull request merged**